### PR TITLE
Add FAQ question for exchange rates

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add FAQ question for exchange rates
| Fixed ticket? | Answers questions like https://github.com/PrestaShop/PrestaShop/issues/23404

### CODE REVIEW

File: src/content/1.7/faq/pricing.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+You also need to provide an authentication `secure_key`.
+
+In order to obtain this secure key, you need 2 items:
+- the `_COOKIE_KEY_` of your shop that you can find in `app/config/parameters.php`
+- your shop name
+
+The secure key is computed like this:
+```
+$secureKey = md5(_COOKIE_KEY_.Configuration::get('PS_SHOP_NAME'));
```
Comment (Progi1984) [inline src/content/1.7/faq/pricing.md:35]:
```suggestion
$secureKey = md5(_COOKIE_KEY_ . Configuration::get('PS_SHOP_NAME'));
```

**Review by Progi1984 — APPROVED** [review]
A small feedback but it's a good job

File: src/content/1.7/faq/pricing.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Refresh exchange rates
+
+**Q:** How can I make sure my currency exchange rates are regularly updated?
+
+**A:** In PrestaShop, all currencies is associated with the exchange rate for Euro.
+
+These exchange rates are updated every day by authorities in charge of these currencies.
+
+PrestaShop gathers these rates and expose them through API endpoint https://api.prestashop.com/xml/currencies.xml (URL available in the code through constant `_PS_CURRENCY_FEED_URL_`).
```
Comment (PierreRambaud) [inline src/content/1.7/faq/pricing.md:17]:
```suggestion
The PrestaShop company gathers these rates and expose them through API endpoint https://api.prestashop.com/xml/currencies.xml (URL available in the code through constant `_PS_CURRENCY_FEED_URL_`).
```

File: src/content/1.7/faq/pricing.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Provided your hosting provider allows you to configure cron jobs, you need to add an entry for a cron job to regularly call the needed URL. [This tool][cron-doc] can help you to write the statement you want.
+
+Example of such a cronjob:
+
+```
+0 6 * * * curl "https://myshop.com/admin_9282/cron_currency_rates.php?secure_key={my_secure_key}"
+```
+
+[crontab]: https://en.wikipedia.org/wiki/Cron
+[cron-doc]: https://crontab.guru/every-1-hour
```
Comment (PierreRambaud) [inline src/content/1.7/faq/pricing.md:55]:
```suggestion
[cron-doc]: https://crontab.guru
```

Comment (PululuK) [inline src/content/1.7/faq/pricing.md:55]:
```suggestion
[cron-doc]: https://en.wikipedia.org/wiki/Cron
[crontab]: https://crontab.guru/every-1-hour
```

Comment (matks) [reply src/content/1.7/faq/pricing.md:55]: I'm not sure to understand 🤔 
- the link `[crontab]` aims to explain what is a crontab (so I redirect to Wikipedia
- the link `[cron-doc]` aims to explain how to add a cron job inside a crontab

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**